controllers/pokemon_controller.go

```diff
@@ -0,0 +1,19 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/rromodev/academy-go-q32021/queries"
+
+	"github.com/gin-gonic/gin"
+)
+
+func GetPokemon(c *gin.Context) {
+	idp := c.Param("id")
+	id, _ := strconv.Atoi(idp)
```

Comment:
Don't ignore these errors, what if the user sends something that isn't a string? If the data from the request is not valid it would be appropriate to return a BadRequest status to the user, along with a message that could go `ID param not valid` or something like that